gdcdictionary/schemas/subject.yaml

```diff
@@ -41,6 +41,7 @@ links:
 required:
   - submitter_id
   - type
+  - unique_de-id_identifier
```

Comment:
making this a required field is a breaking change for all the existing subject data that doesn't have it

Reply:
It was asked to be required by long covid group, is there any other way to make in required ?

Reply:
Not in the dictionary - you will need to make sure that it is always specified on the data ingestion side

Reply:
oh okay, I can do that. Let me make that change and commit

Reply:
I have made the commit with this change.